### ISSUE TITLE
InventorySetupsBankSearch.reset AssertionError

### ISSUE THREAD

**Issue by nbulischeck** [dillydill123/inventory-setups#96]
Just updated my client to 1.7.14 and hit this assertion in developer mode:

```
2021-06-30 09:46:16 [Client] ERROR net.runelite.client.RuneLite - Uncaught exception:
java.lang.AssertionError: Script 299 was called with the incorrect number of arguments; takes 3+0, got 2+0
	at client.uv(client.java:34976)
	at client.runScript(client.java:40802)
	at inventorysetups.InventorySetupsBankSearch.lambda$reset$0(InventorySetupsBankSearch.java:82)
	at net.runelite.client.callback.ClientThread.lambda$invoke$0(ClientThread.java:48)
	at net.runelite.client.callback.ClientThread.invoke(ClientThread.java:61)
	at net.runelite.client.callback.ClientThread.invoke(ClientThread.java:46)
	at inventorysetups.InventorySetupsBankSearch.reset(InventorySetupsBankSearch.java:75)
	at inventorysetups.InventorySetupsPlugin.resetBankSearch(InventorySetupsPlugin.java:753)
	at inventorysetups.InventorySetupsPlugin.onScriptPreFired(InventorySetupsPlugin.java:875)
	at net.runelite.client.eventbus.EventBus$Subscriber.invoke(EventBus.java:66)
	at net.runelite.client.eventbus.EventBus.post(EventBus.java:219)
	at net.runelite.client.callback.Hooks.post(Hooks.java:169)
	at client.if(client.java:41716)
	at df.e(df.java:7830)
	at gk.f(gk.java:109)
	at ai.iv(ai.java:8826)
	at hf.ir(hf.java:38126)
	at client.ju(client.java:7869)
	at client.ef(client.java:3606)
	at client.ae(client.java:1295)
	at aa.du(aa.java:336)
	at aa.run(aa.java:315)
	at java.base/java.lang.Thread.run(Thread.java:832)
```

**Comment by dillydill123** [comment]
Thanks for finding this. In the live client it doesn't appear to be causing any obvious effects. But it looks like the client script has changed somehow. I'll look into it

**Comment by dillydill123** [comment]
Fixed in v1.11
